src/components/Panel/Panel.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import Template from '../Template/Template';
+
+const Panel = ({ cssClasses, hidden, templateProps, data, onRef }) => (
+  <div
+    className={cx(cssClasses.root, {
+      [cssClasses.noRefinementRoot]: !data.canRefine,
+    })}
+    hidden={hidden}
```

Comment:
The `hidden` props and `data.canRefine` feels very similar to me. We can achieve the same goal with both of the options. We can use the API of `hidden` to compute the `canRefine` value, then with CSS we can hide by default the Panel `no-refinement` class. The user is able to override the style applied to the class with a CSS override.

Reply:
So you mean that the `hidden` option could drive the `canRefine` value?

As an example:

```js
panel({
  hidden: () => true, // → cannot refine
})

panel({
  hidden: () => false, // → can refine
})
```

Default rendering options:

```js
{ canRefine: !hidden }
```

I'm not sure to agree here. Hiding and disabling don't solve the same purposes.